### ISSUE TITLE
Dear ZqPei

### ISSUE THREAD

**Issue by silymarins** [ZQPei/deep_sort_pytorch#12]
Hello ZqPei,

I wanted to run train.py.
So I downloaded the dataset Mars and put it as input.
However, there are 625 classes in the train dataset and 636 classes in the test dataset, resulting in a CUDA_error.

So I removed 11 test dataset folders and run the code
Looking at train.jpg, it seems that learning is not working properly
I would appreciate if you could give me an answer.
![image](https://user-images.githubusercontent.com/47780048/54860674-cfcc9080-4d60-11e9-95b0-833d9d8b62b0.png)

**Comment by ZQPei** [comment]
> So I removed 11 test dataset folders and run the code

You shall split validation set from training set of each class.
For example, choose N*2/3 for training and the rest N/3 for validation if there are N images in class 1.
The test set is for evalation.py.

**Comment by silymarins** [comment]
Hello, 
I'm sorry to bother you.
Do you know why the performance of the roadov3 MOTA (29.X) is lower than that of the paper MOTA (61.4)?

**Comment by ZQPei** [comment]
> Do you know why the performance of the roadov3 MOTA (29.X) is lower than that of the paper MOTA (61.4)?

There are mainly two reasons I think,
- 1. In paper, the author uses faster RCNN to generate bounding boxes, which is better than YOLOv3. The reason I use YOLOv3 is that YOLOv3 is much faster than faster RCNN.

- 2. If you uses my parameters in ckpt.t7 , the performance on MOTA may not be good, cause I trained the model on Market1501 not on Mars. This repo is only for demo, and you shall train your own model.

**Comment by silymarins** [comment]
Do you know MOTA, MOTP performance for MOT?

**Comment by ZQPei** [comment]
MOTA: MOT accuracy
MOTP: MOT precision
MOT: a competition
I did not test the performances on MOT dataset.

**Comment by china56321** [comment]
> Hello ZqPei,
> 
> I wanted to run train.py.
> So I downloaded the dataset Mars and put it as input.
> However, there are 625 classes in the train dataset and 636 classes in the test dataset, resulting in a CUDA_error.
> 
> So I removed 11 test dataset folders and run the code
> Looking at train.jpg, it seems that learning is not working properly
> I would appreciate if you could give me an answer.
> ![image](https://user-images.githubusercontent.com/47780048/54860674-cfcc9080-4d60-11e9-95b0-833d9d8b62b0.png)

Hi ,i am training MARS datasets , but the training process is so so so  slow , i trained for 3 days ,just run 20 epoches, the accuracy is 0.3% , Is it normal ?  How did you train it ?
